File: beecrowd -  1060 - Números Positivos.cpp
#include <iostream>
using namespace std;

int main(){
  double a,b,c,d,e,f,soma=0;
  cin >> a >> b >> c >> d >> e >> f;
  if(a>0){
    soma += 1;
  }
  if(b>0){
    soma += 1;
  }
  if(c>0){
    soma += 1;
  }
  if(d>0){
    soma += 1;
  }
  if(e>0){
    soma += 1;
  }
  if(f>0){
    soma += 1;
  }
  printf("%.0f valores positivos\n", soma);

  return 0;
}

File: beecrowd- 1060 - Números Positivos.py
a = 0
l = []
while a < 6:
  x = float(input())
  a += 1
  if x > 0:
    l.append(x)
  else:
    continue
  tamanho = len(l)
print('{} valores positivos'.format(tamanho))


  
